src/Plugins/Nop.Plugin.Shipping.EasyPost/Services/EasyPostService.cs

```diff
@@ -417,17 +417,15 @@ private async Task<Address> PrepareAddressFromAsync(GetShippingOptionRequest req
                 options.currency = EasyPostDefaults.CurrencyCode;
 
             //create shipment
-            var shipment = new Shipment
-            {
-                to_address = addressTo,
-                from_address = addressFrom,
-                parcel = parcel,
-                customs_info = customsInfo,
-                options = options
-            };
+            var shipment = await Shipment.Create();
```

Comment:
This will not work in v3.0.0 of the EasyPost C# library. You need to pass the attributes in a dictionary as a parameter to the `Create` function.

Please refer to the Upgrade Guide here for more details: https://github.com/EasyPost/easypost-csharp/blob/master/UPGRADE_GUIDE.md#30-removes-non-static-create-functions

Reply:
@nwithan8 Thank you for the hint, the shipment creation is now updated.